packages/sit-onyx/src/components/OnyxAppLayout/OnyxAppLayout.stories.ts

```diff
@@ -0,0 +1,125 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxAppLayout from "./OnyxAppLayout.vue";
+
+/**
+ * Layout component that structures your application.
+ * Includes space for a nav bar, page content, and overlays above the page or the whole app.
+ * Recommended to use on App.vue level of an application.
+ */
+const meta: Meta<typeof OnyxAppLayout> = {
+  title: "layout component/OnyxAppLayout",
```

Comment:
```suggestion
  title: "layout/OnyxAppLayout",
```

I would just call it layout to keep it simple :)

Reply:
Same for the OnyxPageLayout